lib/commons/text/label.js

```diff
@@ -8,7 +8,7 @@
 text.label = function (node) {
 	var ref, candidate, doc;
 
-	candidate = aria.label(node);
+	candidate = aria.label(node.actualNode);
```

Comment:
This doesn't seem consistent with the commons file above: `lib/commons/aria/label.js`